src/php/php-agent-download.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+libc=$1
+version=$2
+
+if [ -z "$libc" ]; then
+  echo "'libc' argument not supplied. Defaulting to 'gnu'"
+  libc="gnu"
+elif [[ ! $libc =~ ^(musl|gnu)$ ]]; then
+  echo "error: invalid 'libc' argument provided. Exiting."
+  exit 1
+fi
+
+if [ -z "$version" ]; then
+  latest=$(wget -c https://download.newrelic.com/php_agent/release/ -O - | grep newrelic-php5 | cut -d '-' -f 3 | head -n 1)
+  url="https://download.newrelic.com/php_agent/release/newrelic-php5-${latest}-linux"
+else
+  url="https://download.newrelic.com/php_agent/archive/${version}/newrelic-php5-${version}-linux"
+fi
+
+if [ "$libc" = 'musl' ]; then
+  url+="-musl.tar.gz"
+else
+  url+=".tar.gz"
+fi
+
+wget -c ${url} -O - | tar -xz --strip-components 1
```

Comment:
Can we log errors on the download - for example, if $version was set to an invalid version.

Reply:
Shouldn't `${url}` be quoted?

Reply:
```
shellcheck php-agent-download.sh

In php-agent-download.sh line 27:
wget -c ${url} -O - | tar -xz --strip-components 1
        ^----^ SC2086 (info): Double quote to prevent globbing and word splitting.
````

Reply:
> Can we log errors on the download - for example, if $version was set to an invalid version.

https://github.com/newrelic/newrelic-agent-init-container/pull/84#discussion_r1707806674

Reply:
> Shouldn't `${url}` be quoted?

24425633f88169f4a1cbdb979b4bd1d59baba15c